zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/CamundaExporterIT.java

```diff
@@ -382,4 +334,158 @@ private ExporterResourceProvider mockResourceProvider(
 
     return provider;
   }
+
+  @Nested
+  class ExportHandlerTests {
+
+    @TestTemplate
+    void shouldHandleExportedRecords(
+        final ExporterConfiguration config, final SearchClientAdapter clientAdapter)
+        throws IOException {
+      final var provider = new DefaultExporterResourceProvider();
+      provider.init(config);
+
+      final var exporter =
+          createExporter(
+              new HashSet<>(provider.getIndexDescriptors()),
+              new HashSet<>(provider.getIndexTemplateDescriptors()),
+              config);
```

Comment:
🔧 I guess we could also create the exporter with the provider directly. And call the init/configure and open method here to make more clear that this happens as part of the test as well.